djangoplicity/archives/contrib/info/defaults.py

```diff
@@ -21,6 +20,11 @@
 
 __all__ = ('categories', 'admin_edit', 'admin_edit_for_site', 'admin_add_translation', 'published', 'featured', 'boolean_property', 'priority', 'release_date', 'paper_size', 'link_field')
 
+import django
+if django.VERSION >= (2, 2):
```

Comment:
`django.core.urlresolvers` was removed in django 2.0, therefore the current statement will raise an exception when used in django [2.0, 2.2) range.